src/Microsoft.ML.Mkl.Components/OlsLinearRegression.cs

```diff
@@ -36,12 +36,30 @@ namespace Microsoft.ML.Trainers
     /// ordinary least squares (OLS) for estimating the parameters of the linear regression model.
     /// </summary>
     /// <remarks>
-    /// <a href='https://en.wikipedia.org/wiki/Ordinary_least_squares'>Ordinary least squares (OLS)</a> is a parameterized regression method.
+    /// <format type="text/markdown"><![CDATA[
+    /// To create this trainer, use [Ols](xref:Microsoft.ML.MklComponentsCatalog.Ols(Microsoft.ML.RegressionCatalog.RegressionTrainers, System.String, System.String, System.String))
+    /// or [Ols(Options)](xref:Microsoft.ML.MklComponentsCatalog.Ols(Microsoft.ML.RegressionCatalog.RegressionTrainers, Microsoft.ML.Trainers.OlsTrainer.Options)).
+    ///
+    /// [!include[io](~/../docs/samples/docs/api-reference/io-columns-regression.md)]
+    ///
+    /// ### Trainer Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Machine learning task | Regression |
+    /// | Is normalization required? | Yes |
+    /// | Is caching required? | No |
+    /// | Required NuGet in addition to Microsoft.ML | Microsoft.ML.Mkl.Components |
+    ///
+    /// ### Training Algorithm Details
+    /// [Ordinary least squares (OLS)](https://en.wikipedia.org/wiki/Ordinary_least_squares) is a parameterized regression method.
     /// It assumes that the conditional mean of the dependent variable follows a linear function of the dependent variables.
-    /// The regression parameters can be estimated by minimizing the squares of the difference between observed values and the predictions.
+    /// The regression parameters can be estimated by minimizing the squares of the difference between observed values and the predictions
+    /// ]]>
+    /// </format>
     /// </remarks>
-    /// <seealso cref="MklComponentsCatalog.Ols(RegressionCatalog.RegressionTrainers, string, string, string)"/>
-    /// <seealso cref="MklComponentsCatalog.Ols(RegressionCatalog.RegressionTrainers, OlsTrainer.Options)"/>
+    /// <seealso cref="Microsoft.ML.MklComponentsCatalog.Ols(Microsoft.ML.RegressionCatalog.RegressionTrainers, System.String, System.String, System.String)"/>
```

Comment:
> System.String, System.String, System.String) [](start = 111, length = 45)

no need to change these to fully qualified. cref is more forgiving. whatever link visual studio intellisense gives you should be good.